crypto/crypto.go

```diff
@@ -197,12 +165,15 @@ func NewNaiveSigner(privateKey PrivateKey, hashAlgo HashAlgorithm) NaiveSigner {
 	return NewInMemorySigner(privateKey, hashAlgo)
 }
 
+// targeted security bits of the cryptographic algorithms
+const securityBits = 128
+
 // MinSeedLength is the generic minimum seed length required to guarantee sufficient
 // entropy when generating keys.
 //
 // This minimum is used when the seed source is not necessarily a CSPRG and the seed
 // should be expanded before being passed to the key generation process.
-const MinSeedLength = crypto.MinSeedLen
+const MinSeedLength = 2 * (securityBits / 8)
```

Comment:
Maybe this should be a constant in flow-go/crypto?

Reply:
The crypto lib defines other types of minimum seed lengths because it makes other assumptions about the seed quality.
This constant is here to address another point when the SDK generates keys from passphrases.